gsa/src/gmp/models/__tests__/scanner.js

```diff
@@ -51,6 +56,11 @@ describe('Scanner model tests', () => {
     expect(scanner.scannerType).toEqual(42);
     expect(scanner2.scannerType).toEqual(1);
     expect(scanner3.scannerType).toEqual(9);
+
+    // Hyperion
+    const scanner4 = Scanner.fromObject({type: 'OSP_SCANNER_TYPE'});
+
+    expect(scanner4.scannerType).toEqual(1);
```

Comment:
We should always compare such values to some constant variable like OSP_SCANNER_TYPE to be able to change the meaning of the value e.g. use a string or a different int in future.